gradle/strict.gradle

```diff
@@ -22,7 +22,7 @@
 
 tasks.withType(JavaCompile) {
     // suppress static due to protoc generating super.addAll calls.
-    options.compilerArgs += ["-Xlint:all", "-Xlint:-processing", "-Xlint:-options", "-Xlint:-static", "-Werror"]
```

Comment:
Was this an accident. I believe error-on-warn is part of the definition of "strict."

Reply:
@MMcM ah sorry this is not supposed to be committed. umm the problem is that it reports warning as error because the code in ```QueryPlanUtils.getIndexEntryFDBQueriedRecordFunction()``` casts the ```toRecord.toRecord(recordDescriptor, indexEntry)``` to ```M```, which was existing code and I just refactored it for reuse.

Reply:
Reverted this accidental change and suppressed the warning on that method.